### PR TITLE
Feat/small visual fixes

### LINKED ISSUE #83: Small visual fixes
## Short overview
Task aggregating small fixes & improvements

## ToDo

- [x] Padding on mobile menu should be higher
image
- [x] Table of content can be hidden for now
image
- [x] Fix edit this page on index page
image
- [x] Add `Getting Started` item at the top of the menu without section. It should redirect to main page
- [x] Paragraphs have wrong HTML
image
- [x] investigate  (might be the parser issue ) wrong HTML (avatar component)

## Contact points
@k1eu

### PR DESCRIPTION
**Pull request by DawidHaratyk**
# Overview:

closes #83

Short description and worth mentioning decision made.

- After parsing to HTML in preview, I've replaced all wrongly closing tags (input, img)

### CODE REVIEW

**Comment by PoslinskiNet** [comment]
@k1eu I would leave it as it is. We can change it later, and the changes already bring value to the production release. During the journey with the development of this design system, we will change it probably few times, so I suggest not spending too much time on that for now.

File: packages/selleo-design-core/src/parser.tsx [hunk 1]
```diff
@@ -16,10 +16,11 @@ const buildComponent = (html: string) => (): VNode => {
 export const parseComponent = (html: string) => {
   const Component = buildComponent(html);
   const HTMLString = render(<Component />, null, { pretty: true });
+  const HTMLStringResult = HTMLString.replaceAll(/><\/input>|><\/img>/g, '>');
```
Comment (k1eu) [inline packages/selleo-design-core/src/parser.tsx:19]: nice, GJ with it! :D 
Can we also replace in ReactString to be self-closing?

**Review by k1eu — CHANGES_REQUESTED** [review]

File: packages/selleo-design-core/src/parser.tsx (hunk 1 above)
Comment (DawidHaratyk) [reply packages/selleo-design-core/src/parser.tsx:19]: Thanks! 😁
Yes, already changed :)

File: apps/docs/src/layouts/MainLayout.astro [hunk 2]
```diff
@@ -17,13 +17,13 @@ type Props = {
 const { frontmatter, headings } = Astro.props as Props;
 const canonicalURL = new URL(Astro.url.pathname, Astro.site);
 const currentPage = Astro.url.pathname;
-const currentFile = `src/pages${currentPage.replace(/\/$/, "")}.mdx`;
+const currentFile = `src/pages${currentPage.replace(/\/$/, '')}`;
```
Comment (k1eu) [inline apps/docs/src/layouts/MainLayout.astro:20]: with that change neither main page `edit` works nor the components edit pages. Please fix

File: packages/selleo-design-core/src/parser.tsx [hunk 3]
```diff
@@ -17,9 +17,11 @@ export const parseComponent = (html: string) => {
   const Component = buildComponent(html);
   const HTMLString = render(<Component />, null, { pretty: true });
   const ReactString = parseToReact(HTMLString);
+  const HTMLStringResult = HTMLString.replaceAll(/><\/input>|><\/img>/g, '>');
+  const ReactStringResult = ReactString.replaceAll(/><\/input>|><\/img>/g, '>');
```
Comment (k1eu) [inline packages/selleo-design-core/src/parser.tsx:21]: That's the same as HTML, with React we would like to have the proper react self closing elements `<element />` not `<element>`

**Review by k1eu — CHANGES_REQUESTED** [review]

File: packages/selleo-design-core/src/parser.tsx (hunk 3 above)
Comment (k1eu) [reply packages/selleo-design-core/src/parser.tsx:21]: @DawidHaratyk please add this small change and it will be approved :>

**Comment by PoslinskiNet** [comment]
Do we need this edit button at all?

File: apps/docs/src/components/LeftSidebar/LeftSidebar.astro [hunk 4]
```diff
@@ -15,9 +15,9 @@ const langCode = getLanguageFromURL(currentPage);
 const sidebar = SIDEBAR[langCode];
 ---
 
-<nav class="mr-2 w-full" aria-labelledby="grid-left">
+<nav class="mr-2 mt-2 w-full" aria-labelledby="grid-left">
```
Comment (k1eu) [inline apps/docs/src/components/LeftSidebar/LeftSidebar.astro:18]: please remove mt-2 on resolutions without mobile menu so the getting statrted is aligned with the most top conent (`introduction`)

**Review by k1eu — APPROVED** [review]
Approved, one very small not breaking change so merge after fixing it. Just make sure the commits make sense ;)

File: apps/docs/src/components/LeftSidebar/LeftSidebar.astro (hunk 4 above)
Comment (DawidHaratyk) [reply apps/docs/src/components/LeftSidebar/LeftSidebar.astro:18]: Okay, done

**Pull request merged**